docs/usage/administration/admin_api/README.md

```diff
@@ -5,9 +5,19 @@
 Many of the API calls in the admin api will require an `access_token` for a
 server admin. (Note that a server admin is distinct from a room admin.)
 
-A user can be marked as a server admin by updating the database directly, e.g.:
+An existing user can be marked as a server admin by updating the database directly.
 
+When using PostgreSQL, switch to the `postgres` user and start the command line tool.
```

Comment:
I'm not sure this is accurate -- not everyone will have a `postgres` user to switch to. I think this is why it was left a bit agnostic (and isn't copy & paste directions).

Reply:
Yes, to me personally it would have been helpful to have this example to understand what is meant. As it was left agnostic, I could not follow the steps immediately. In other words, I would have preferred to have this information.
I followed the installation guide and pretty quickly switched from SQLite to PostreSQL. I do not know how to do this in SQLite, which is why I am starting the sentence with "When using PostreSQL...". Maybe eventually someone else will extend the instructions for SQLite...

Reply:
I mean that these directions don't make sense if someone is using postgres on a different system or didn't install it as the postgres user (maybe they're using Docker).

Reply:
Ok. Though I do not have the knowledge about all different kinds of setups to extend the document accordingly. I guess we could start with this information and extend it eventually?! I could prepend an "For example:" for everyone to know that this might not apply to them.

Reply:
I agree with @clokep here.

We want to keep it generic here so I would at least drop the part about the `postgres` user and just mention something like "Connect to the correct database using either `psql [database name]` (if using PostgreSQL) or `sqlite3 path/to/your/database.db` (if using SQLite)". This would also allow dropping the `\c synapse` line below (since it's not needed when you tell `psql` which database to use directly).

Reply:
Also worth noting that you could just connect using the same details as you have configured in Synapse's config, whatever they may be.